tests/nvmeof/test_ceph_nvmeof_gateway.py

```diff
@@ -16,6 +16,8 @@
 
 LOG = Log(__name__)
 
+# def run_cli_with_check(gw, *args, **kwargs):
```

Comment:
I believe this comment is here by mistake. We could remove as we have not implemented this